Description: expose old method names for backward compatibility
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2018-06-08
Forwarded: not-needed
--- a/index.js
+++ b/index.js
@@ -1,11 +1,11 @@
 'use strict'
 
 // high-level commands
-exports.c = exports.create = require('./lib/create.js')
-exports.r = exports.replace = require('./lib/replace.js')
-exports.t = exports.list = require('./lib/list.js')
-exports.u = exports.update = require('./lib/update.js')
-exports.x = exports.extract = require('./lib/extract.js')
+exports.c = exports.create = exports.Create = require('./lib/create.js')
+exports.r = exports.replace = exports.Replace = require('./lib/replace.js')
+exports.t = exports.list = exports.List = require('./lib/list.js')
+exports.u = exports.update = exports.Update = require('./lib/update.js')
+exports.x = exports.extract = exports.Extract = require('./lib/extract.js')
 
 // classes
 exports.Pack = require('./lib/pack.js')
